Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,36 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            // Homework Lesson 1 & 2 - SilentKingdom
+            // BMI Calculator for 2 people
+
+            // First person = variable1
+            string name1 = "Tom Jefferson";
+            int age1 = 19;
+            float weight1 = 50f;
+            float height1 = 156.5f;
+            // Testing string + variable layouts
+            Console.WriteLine("{0} is {1}, and weighs {2}kg at a height of {3}cm.", name1, age1, weight1, height1);
+
+            // Their BMI
+            float heightConvert1 = height1 / 100;
+            float height_s1 = heightConvert1 * heightConvert1;
+            float sum1 = weight1 / height_s1;
+            Console.WriteLine(name1 + " has a BMI of " + sum1 + ".");
+
+
+            // Second person = variable2
+            string name2 = "Constance Derpleton";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'name2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4832834)